src/App.js

```diff
@@ -1,10 +1,14 @@
-import React from 'react';
-import './App.css';
-import { Home, About, Contact, Navigation } from './components';
+import React from "react";
+import { Route } from "react-router-dom";
+import "./App.css";
+import { Home, About, Contact, Navigation } from "./components";
 
 const App = () => (
   <div>
     <Navigation />
+    <Route exact path="/" component={Home} />
+    <Route path="/about" component={About} />
+    <Route path="/contact" component={Contact} />
```

Comment:
good catch in not unnecessarily adding the `exact` attribute to About or Contact since with the current routes, they could never be simultaneously rendered via the Router